tests/testthat/test-generate_qc_report.R

```diff
@@ -5,8 +5,23 @@ filt_sce <- filter_counts(sce)
 test_that("generating a qc report works", {
   qc_file <- generate_qc_report(library_id = "TEST",
                                 unfiltered_sce = sce,
-                                filtered_sce = filt_sce, )
+                                filtered_sce = filt_sce)
   expect_true(file.exists(qc_file))
   # clean up
   file.remove(qc_file)
 })
+
+test_that("generating a qc report does not work", {
+  expect_error(generate_qc_report(library_id = "TEST",
+                                unfiltered_sce = sce,
+                                filtered_sce = filt_sce,
+                                rmd_file = "test.rmd"))
```

Comment:
Just for a bit of extra clarity?
```suggestion
                                rmd_file = "missing.rmd"))
```

Reply:
I think we probably also want a tests that _does_ render a report with a template. That test template can probably be extremely short, but it should exist...